### PR TITLE
Add "Enable Pack Release" Workflow

### PR DESCRIPTION
**Pull request by mamercad**
Related to [this](https://github.com/StackStorm-Exchange/ci/pull/137).

### CODE REVIEW

File: .github/workflows/update-index.yaml [hunk 1]
```diff
@@ -18,6 +18,11 @@ on:
   workflow_dispatch:
 
 jobs:
+
+  enable_pack_release_workflow:
+    name: 'Enable Pack Release Workflow'
+    uses: StackStorm-Exchange/ci/.github/workflows/pack-enable_release_workflow.yaml@master
```
Comment (cognifloyd) [inline .github/workflows/update-index.yaml]: Doing this every 5 minutes with the index update workflow is a bit excessive.
Could we run this job in a separate workflow once a month?

Comment (mamercad) [reply .github/workflows/update-index.yaml]: > Doing this every 5 minutes with the index update workflow is a bit excessive. Could we run this job in a separate workflow once a month?

Hehe, totally.

**Comment by cognifloyd** [comment]
I wonder if it would make more sense to schedule this workflow on https://github.com/StackStorm-Exchange/ci instead of on the index. The current workflow in this repo only has write access to the index repo, and nothing else. I think I'd like to keep it that way.

What do you think?

**Comment by mamercad** [comment]
> I wonder if it would make more sense to schedule this workflow on https://github.com/StackStorm-Exchange/ci instead of on the index. The current workflow in this repo only has write access to the index repo, and nothing else. I think I'd like to keep it that way.
> 
> What do you think?

Yeah, that probable makes more sense actually :loading:

**Comment by mamercad** [comment]
Closing in favor or [this](https://github.com/StackStorm-Exchange/ci/pull/137).

**Pull request closed without merging**